infrastructure/preparation/projet_doctoral/domain/service/profil_candidat.py

```diff
@@ -56,7 +60,30 @@ def get_identification(cls, matricule: str) -> 'IdentificationDTO':
 
     @classmethod
     def get_coordonnees(cls, matricule: str) -> 'CoordonneesDTO':
-        pass
+        domicile_legal = PersonAddress.objects.select_related('country').filter(
+            person__global_id=matricule,
+            label=PersonAddressType.RESIDENTIAL.name,
+        ).first()
+
+        adresse_correspondance = PersonAddress.objects.select_related('country').filter(
+            person__global_id=matricule,
+            label=PersonAddressType.CONTACT.name,
+        ).first()
```

Comment:
Deux requêtes SQL :/

```suggestion
        # TODO Use in_bulk() in Django 3.2
        adresses = {
            a.label: a
            for a in PersonAddress.objects.select_related('country').filter(
                person__global_id=matricule,
                label__in=[PersonAddressType.CONTACT.name, PersonAddressType.RESIDENTIAL.name],
            )
        }    
        domicile_legal = addresses.get(PersonAddressType.RESIDENTIAL.name)
        adresse_correspondance = addresses.get(PersonAddressType.CONTACT.name)
```